data/map planets.txt

```diff
@@ -3316,6 +3365,10 @@ planet "New Austria"
 	description `	For centuries the locals have been trying to find a deposit of diamonds, which would sell for much higher prices than sapphires, and local folklore revolves around the prosperity that would come to this world if diamonds were ever discovered.`
 	spaceport `The spaceport is nothing but a couple of landing pads on the outskirts of one of the larger mining villages, high up on a mountain ridge to allow easy access for starships. A few roads lead down from there to the village, where you can trade for local goods or get food from the miners' cafeteria.`
 	spaceport `	By the time you have walked to the center of the town, several people have already tried to sell you large "diamonds" that you strongly suspect are just clear sapphires.`
+	spaceport `	The village itself is abuzz for Remembrance Day. Various exhibits have been set up, displaying either replicas of the planet's largest sapphires, or the real deals. You recall seeing a cheesy Remembrance Day movie about a heist for one such sapphire when you were younger.`
```

Comment:
So what about other starts?
Would this still be the case?

Reply:
Just append another condition to take into account the start.